cart.html

```diff
@@ -0,0 +1,125 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="index.css" />
+     <link
+      rel="stylesheet"
+      href="https://use.fontawesome.com/releases/v5.5.0/css/all.css"
+      integrity="sha384-B4dIYHKNBt8Bc12p+WXckhzcICo0wtJAoU8YZTY5qE0Id1GSseTk6S+L3BlXeVIU"
+      crossorigin="anonymous"
+    />
+    <title>NeoDev Store</title>
+  </head>
+  <body>
+    <div class="navigation_main_container">
+      <div class="nav_section">
+        <div class="burger">
+          <i class="fas fa-bars"></i>
+        </div>
+        <div class="brand_logo">
+          <a href="/">
+            <img src='./images/logo.png' alt="" />
+          </a>
+        </div>
+        <div class="brand_title">
+          <a href="/">NeoDev</a>
+        </div>
+      </div>
+  
+      <div class="nav_section">
+        <div class="nav_searchbar">
+          <span>
+            <i class="fa fa-search fa"></i>
+          </span>
+          <input type="text" placeholder="Type to search" />
+        </div>
+        <div class="nav_icons_container">
+          <div class="nav_icon profile_icon">
+            <a href="#" class="nav_icon">
+                <span>
+                  <i class=" fas fa-user-circle fa"></i>
+                </span>
+                <span class="nav_icon_name">Profile</span>
+            </a>
+          </div>
+          <div class="nav_icon">
+            <a href="/cart.html" class="nav_icon">
+                <span>
+                  <i class="cart_icon fas fa-shopping-cart fa"></i>
+                </span>
+  
+                <span class="nav_icon_name">Cart</span>
+            </a>
+          </div>
+          <div class="nav_icon">
+            <a href="#" class="nav_icon">
+                <span>
+                  <i class=" wishlist_icon fas fa-heart fa"></i>
+                </span>
+  
+                <span class="nav_icon_name">Wishlist</span>
+            </a>
+          </div>
+        </div>
+      </div>
+    </div>
+    <div class="cart_container">
+      <h1>Cart</h1>
+
+      <div class="cart_main_container">
+        <div class="cart_item">
+          <a href="/productlist">
+            <img class="img_responsive" src="https://images.pexels.com/photos/7319044/pexels-photo-7319044.jpeg?auto=compress&cs=tinysrgb&dpr=3&h=750&w=1260" alt=""></img>
+          </a>
+
+          <div class="item_desc">
+            <h3 class="item_desc_heading">Black Tshirt for Women</h3>
+            <p>Tshirt</p>
+            <p>$30</p>
+  
+            <button class="btn" type="submit">
+              Remove from cart
+            </button>
+            <button class="btn secondary_outline_btn" type="submit">
+              Move to wishlist
+            </button>
+          </div>
+        </div>
+
+      
+
+      <div class="price_details_container">
```

Comment:
sugesstion: for listing price and value  ul tag can be used , easier and less css

Reply:
Okay will try that later.